### PR TITLE
Fix ug bugs

### LINKED ISSUE #194: [PE-D] include command (documentation and bugs)
The documentation is incomplete; I was quite lost as to what "INFORMATION" meant. 

Also, when I passed in commands that are according to the examples, it results in this error. The resident "A429" is the resident that i input into the resident portion.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: Domszy/ped#4

### LINKED ISSUE #201: [PE-D] No indication on the baseline for labelling FET as late.
When using the app, I noticed that 22 Oct 2021 is the benchmark to determine whether FET is late.
It is not clear (or not obvious) in the UG that perhaps the previous Friday is the benchmark? 

More information needs to be given to make this clear to users.

-------------
Labels: `type.DocumentationBug` `severity.High`
original: eeliana/ped#16

### LINKED ISSUE #210: [PE-D] Suggestion for `deadline`
Perhaps some screenshots of how this feature work could be added here? Also, can consider defining what is `due on` and `collected before` as definition is not stated, perhaps it would make it clearer for the readers. 
The description given might not be clear enough for readers to understand how this feature actually work. 

Personally, after reading through the description and examples, it is still quite confusing for me to understand this feature.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: Py0000/ped#11

### PR DESCRIPTION
**Pull request by gordonlzy**
Closes #194 
Closes #201 
Closes #210

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -387,12 +393,12 @@ Examples:
...
-Format: `include INDEX r/INFORMATION [, MORE INFORMATION]`
+Format: `include INDEX [r/RESIDENTS]`
 
-* Resident information can be given in the form of name or room, but all has to be all rooms or all names
-* When adding multiple residents, each piece of information is separated by a comma
-* The information inputted is case-insensitive
-* If one or more of the given information is invalid, an error message is outputted and none of the residents are added to the event
+* Residents can be given in the form of name or room, but all has to be all rooms or all names
+* When adding multiple residents, each resident is separated by a comma
+* The resident inputted is case-insensitive
```
Comment (chongjunwei) [inline docs/UserGuide.md:400]: Perhaps could be "names/rooms" instead of resident for clarity

File: docs/UserGuide.md [hunk 2]
```diff
@@ -387,12 +393,12 @@ Examples:
...
+* If one or more of the given resident is invalid, an error message is outputted and none of the residents are added to the event
```
Comment (chongjunwei) [inline docs/UserGuide.md:401]: Same here

File: docs/UserGuide.md [hunk 3]
```diff
@@ -145,17 +145,23 @@ Examples:
...
+Lists residents whose ART collection or FET tests are either:
+* due before a given date, `d1`, by using the `late keywords` or 
+* due within the range of two given dates, `d1` and `d2`, by using the `normal keywords`.
+
+There is a one week deadline for the test kit collection and fet test, therefore due date refers to one week after the last recorded date
 
 Format: `deadline k/KEYWORD d1/DATE1 d2/DATE2` or `deadline k/LATE_KEYWORD d1/DATE1`
 
+* A resident's fet or collection is due one week after their last fet date or last collection date. For example, if a resident's last fet date is on a friday, `15-10-2021`, then the resident's fet deadline is on the following friday which is `22-10-2021`
+* The number of days a resident is considered late is calculated from `a day after` their deadline to the `current date`, both inclusive
```
Comment (chongjunwei) [inline docs/UserGuide.md:157]: Can add an example here for more clarity

**Review by chongjunwei — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 3 above)
Comment (gordonlzy) [reply docs/UserGuide.md:157]: Good idea

File: docs/UserGuide.md (hunk 2 above)
Comment (gordonlzy) [reply docs/UserGuide.md:401]: okay

File: docs/UserGuide.md (hunk 1 above)
Comment (gordonlzy) [reply docs/UserGuide.md:400]: sounds good

**Review by VimuthM — CHANGES_REQUESTED** [review]
Can the exclude documentation be updated as well similarly? For example, remove `INFORMATION`

**Review by VimuthM — APPROVED** [review]

**Pull request merged**